### ISSUE TITLE
"Warning: Accessing PropTypes via the main React package is deprecated" for React 15.5+

### ISSUE THREAD

**Issue by lahmatiy** [algolia/react-element-to-jsx-string#213]
With commit https://github.com/algolia/react-element-to-jsx-string/commit/73d60526345e06f433929a7485abe1638e2da3da some import were changed to:

```js
import * as React from 'react';
```

In this case babel uses `_interopRequireWildcard` which is using `for..in` to make a copy of module exports. This cause to warning when React 15.5+ is using:

```
Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
```

Because with React 15.5, importing PropTypes from React is deprecated in favour of importing from the new 'prop-types' package.

**Comment by Spy-Seth** [comment]
This king of import is how flow advice to do to work with React: https://flow.org/en/docs/react/types/

This will not be an issue with React 16. And I'm not sure we could do something about it 😕  It's look like a babel issue. 

If someone have a suggestion I'm open 😅

**Comment by lahmatiy** [comment]
Issue fixed by PR #224
